lib/chef/provider/mount/mount.rb

```diff
@@ -267,6 +246,35 @@ def mount_options_unchanged?
             @current_resource.pass == @new_resource.pass
         end
 
+        # It will update or delete the entry from fstab.
+        def update_or_delete_fs(update_fs = false)
```

Comment:
If we're going to go down the route of having a helper that actually does the work it should just be `update_fstab` and takes an action value. We've done that same sort of thing in other places. When you see how this is used right now it's not clear what the parameter is actually doing unless you look at the definition

Reply:
It becomes something like this

```ruby
edit_fstab(:remove)
```

or 

```ruby
edit_fstab(:update)
```

Reply:
@tas50 Agreed, I have updated it. Please have a look.